css/giving-rides.css

```diff
@@ -0,0 +1,304 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+body,
+h1,
+h2,
+h3,
+h4,
+p,
+ul[class],
+ol[class],
+li,
+figure,
+figcaption,
+blockquote,
+dl,
+dd {
+  margin: 0;
+}
+
+html{
+    /* font-size:62.5%; */
+    text-decoration: none;
+    font-family: 'Open Sans', sans-serif;
+}
+ul{
+    list-style-type: none;
+    margin: 0;
+}
+
+.important-info{
+    width: auto;
+    height: 8.5rem;
+    background-color: #DFF4F3;
+    font-size: 2.6rem;
+    display: flex;
+    align-items: center;
+    padding-left:11rem; 
+}
+#info-list li::before{
+    content: "\25A0";
+    color: #5C6E93;
+    display: inline-block; /* Needed to add space between the bullet and the text */
+    width: 5rem; /* Also needed for space (tweak if needed) */
+    padding-top: 2rem;
+    margin-left: -5rem; /* Also needed for space (tweak if needed) */
+}
+#info-list{
+    margin-top: 2rem;
+    display:flex;
+    /* list-style-type: square; */
+    flex-direction:column;
+    font-size: 1.6rem;
+    margin-left: 37rem;
+    /* margin-right: 35rem; */
+}
+.info-icon-list{
+    margin-top: 10rem;
+    margin-right: 35rem;
+    margin-left: 35rem;
+    padding: 0;
+    font-size: 1.6rem;
+    display:flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+.info-icon-list-home{
+    margin-top: 10rem;
+    margin-right: 27rem;
+    margin-left: 27rem;
+    padding: 0;
+    font-size: 1.6rem;
+    display:flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+.info-icon-box{
+    text-align: center;
+    color: #807C7C;
+    background-color: #F1F1F1;
+    width: 30%;
```

Comment:
You can set a min-width on all of the boxes so that you don't need to update them responsively with @ media queries